src/lib/modernExtend.ts

```diff
@@ -1,16 +1,20 @@
 import {Zcl} from 'zigbee-herdsman';
 import tz from '../converters/toZigbee';
 import fz from '../converters/fromZigbee';
-import {Fz, Tz, ModernExtend, Range, Zh, Logger, DefinitionOta, OnEvent, Access} from './types';
```

Comment:
It's hard to review the diff of this file, can you summarise  what you changed here? You just added more modernExtends and re-ordered them?

Reply:
Yes, the size was getting out of hands. I almost added pressure extend twice. They were sorted roughly according to ZCl documentation chapters. All additions and fixes are listed as todos. Mostly just new extends

Reply:
Would it make sense to break this up into a separate file per ZCL chapter and maybe 1 file for the custom stuff that is not in the ZCL ?

Reply:
There are not a lot of generic extends left to add (switch configuration and hvac I think) so I don’t see a point in splitting them any further. All manufacturer specific stuff is in their files anyway